Lombiq.HelpfulLibraries/Libraries/Mvc/TypedRoute.cs

```diff
@@ -86,17 +95,20 @@ private string GetRoute(string route)
 
         public static TypedRoute CreateFromExpression<TController>(
             Expression<Action<TController>> actionExpression,
-            IEnumerable<(string Key, object Value)> additionalArguments,
+            IList<(string Key, object Value)> additionalArguments,

```

Comment:
Huh, thanks for catching that. It's not actually needed any more.

Originally there was multiple enumeration in the other overload so I switched that one to IList to avoid that. I switched this one to IList to retain parity becuase one overload as IList and the other as Enumerable would've been confusing. I found a different approach that's single enumeration so I will switch it back to IEnumeration.